src/EnergyPlus/CMakeLists.txt

```diff
@@ -570,11 +570,16 @@ install( TARGETS energyplusapi
   LIBRARY DESTINATION ./ 
 )
 
-if (CMAKE_COMPILER_IS_GNUCXX AND ENABLE_LTO)
+if (CMAKE_COMPILER_IS_GNUCXX)
```

Comment:
This seems probably OK. I had not found that the cross-linking of energypluslib and energypluslib2 was necessary on g++ in the general case, so I avoided it because it adds an extra dependency and slows down the build.